common/translations/lang/en.json

```diff
@@ -655,6 +655,7 @@
     "NETWORK": "Network",
     "NETWORK_2": "network",
     "PROVIDED_BY": "provided by",
-    "YOU_ARE_INTERACTING": "You are interacting with the"
+    "YOU_ARE_INTERACTING": "You are interacting with the",
```

Comment:
@wbobeirne Previously had some comments to address these strings that made their way during the ducks refactor. We don't have to do it in this PR, but just a reminder.

Reply:
https://github.com/MyCryptoHQ/MyCrypto/issues/1981 captures this

Reply:
👍

Reply:
Hey